drivers/sensors/pimoroni_trackball.h

```diff
@@ -20,6 +20,12 @@
 #include "report.h"
 #include "i2c_master.h"
 
+#ifdef MOUSE_EXT_REPORT
+typedef int32_t clamp_range_t;
+#else
+typedef int16_t clamp_range_t;
+#endif
+
```

Comment:
Aren't these already in `pointing_device.h`? Seems little reason to duplicate... can they be removed?